ldap/3.5/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,553 @@
+
+* Access to an OpenLDAP instance that supports LDAP v3.
+* Anypoint Studio version 7.0 (or higher) or Anypoint Design Center.
+
+To use the LDAP Connector in a production environment, you must have either:
+
+* An Enterprise license to use Mule.
+* A CloudHub Starter, Professional, or Enterprise account.
+
+To use this information, you should be familiar with LDAP, Mule runtime engine (Mule), Anypoint Connectors,
+Anypoint Studio essentials, elements in a Mule flow, and global elements.
+
+== POM File Information
+
+[source,xml,linenums]
+----
+<dependency>
+  <groupId>com.mulesoft.connectors</groupId>
+  <artifactId>mule-ldap-connector</artifactId>
+  <version>x.x.x</version>
+  <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+Replace `x.x.x` with the version that corresponds to the connector you are using.
+
+To obtain the most up-to-date `pom.xml` file information:
+
+. Go to https://www.mulesoft.com/exchange/[Anypoint Exchange].
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. In Exchange, search for `ldap`.
+. Select the connector.
+. Click *Dependency Snippets* near the upper right of the screen.
+
+[[compatibility]]
+=== Compatibility
+
+LDAP Connector is compatible with:
+
+[%header,cols="20a,80a",width=70%]
+|===
+|Application/Service|Version
+|Mule Runtime|4.1.1 and later
+|Java|1.8.0_x or higher
+|Anypoint Studio|7.0 or higher
+|LDAP| v3
+|OpenLDAP| 2.4.x
+|===
+
+[[install]]
+== Add the Connector to a Studio Project
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. In Studio, create a Mule project.
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In the *Add Dependencies to Project* window, type `ldap` in the search field.
+. Click *LDAP Connector* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[config]]
+== Configure the LDAP Connector Global Element
+
+To use LDAP Connector in a Mule application, you must configure a global LDAP configuration element that can be used by as many LDAP Connectors as you require for your application. This connector supports server side pooling for Basic configuration, which can be configured in the Pooling section.
+
+. Click the *Global Elements* tab at the base of the canvas.
+. On the Global Configuration Elements screen, click *Create*. +
+The following window appears:
++
+image::ldap-config-global-wizard.png[Global Element Configuration Wizard]
++
+. In the Choose Global Type wizard, expand *Connector Configuration*, select *LDAP Configuration*, and click *OK*. +
+The following window appears:
++
+image::ldap-configs.png[Global Element Properties]
++
+. Choose Basic configuration or TLS configuration from the drop-down list and configure its properties.
```

Comment:
```suggestion
. Choose Basic configuration, SSL configuration or TLS configuration from the drop-down list and configure its properties.
```

Reply:
Also, both SSL and TLS configurations now have a security configuration that is optional:
![Screen Shot 2022-02-08 at 14 33 08](https://user-images.githubusercontent.com/41025937/153043183-569362a5-7f88-450f-930e-59426332779d.png)